i18n/qqq.json

```diff
@@ -353,5 +353,19 @@
 	"srf-spreadsheet-link": "{{doc-smw-link}}",
 	"srf-paramdesc-spreadsheet-filename": "{{doc-paramdesc|filename}}",
 	"srf-paramdesc-spreadsheet-templatefile": "{{doc-paramdesc|templatefile}}",
-	"srf-paramdesc-icalendar-timezone": "{{doc-paramdesc|timezone}}"
+	"srf-paramdesc-icalendar-timezone": "{{doc-paramdesc|timezone}}",
+	"srf-paramdesc-gantt-diagramtitle": "{{doc-paramdesc|diagram}}",
+	"srf-paramdesc-gantt-diagramtheme": "{{doc-paramdesc|theme}}",
+	"srf-paramdesc-gantt-axisformat": "{{doc-paramdesc|axisformat}}",
+	"srf-paramdesc-gantt-sortkey": "{{doc-paramdesc|sortkey}}",
+	"srf-paramdesc-gantt-titletopmargin": "{{doc-paramdesc|titletopmargin}}",
+	"srf-paramdesc-gantt-barheight": "{{doc-paramdesc|barheight}}",
+	"srf-paramdesc-gantt-leftpadding": "{{doc-paramdesc|leftpadding}}",
+	"srf-paramdesc-gantt-bargap": "{{doc-paramdesc|bargap}}",
+	"srf-error-gantt-theme" : "This is an error message.",
+	"srf-error-gantt-sortkey" : "This is an error message.",
+	"srf-error-gantt-mapping-assignment": "This is an error message.",
+	"srf-error-gantt-mapping-keywords": "This is an error message.",
+	"srf-printername-gantt": "{{doc-smwformat|vcard}}"
```

Comment:
I guess this is meant to be gantt

Reply:
Commit-ID: 8f01390aeb3ff92489fa8758f44a1b7d593d8f8a